client/src/components/PhotoModal.jsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import Photo from './Photo.jsx';
+import { FaChevronRight } from 'react-icons/fa';
+import { FaChevronLeft } from 'react-icons/fa';
+import { FaTimes } from 'react-icons/fa';
+
+class PhotoModal extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      currentPhoto: 0,
+    };
+    this.handleRightArrowClick = this.handleRightArrowClick.bind(this);
+    this.handleLeftArrowClick = this.handleLeftArrowClick.bind(this);
+  }
+
+  handleLeftArrowClick() {
+    if (this.state.currentPhoto > 0) {
+      this.setState({
+        currentPhoto: this.state.currentPhoto -= 1,
+      });
+    } else {
+      this.setState({
+        currentPhoto: this.state.currentPhoto = this.props.photoDetails.length - 1,
```

Comment:
Woah! What's going on here with that `=` in the `setState` here?  Please revisit this.